.github/workflows/test-workflow.yml

```diff
@@ -0,0 +1,11 @@
+- name: Check PR title
```

Comment:
Could you remove this file. It was not due to you but some event on `main`.

Reply:
Removed, thanks for noting.